### PR TITLE
WT-5928 Cleanup stale FIXMEs from durable history

### PR DESCRIPTION
**Pull request by tetsuo-cpp**
This PR is to:
- Remove stale FIXMEs.
- Tag existing FIXMEs with a more relevant PM or WT ticket (a lot were tagged with the "write newest" project).
- Tag existing FIXMEs with a new WT ticket.

### CODE REVIEW

File: src/include/cell.i [hunk 1]
```diff
@@ -470,7 +470,7 @@ __wt_cell_pack_del(WT_SESSION_IMPL *session, WT_CELL *cell, wt_timestamp_t start
     p = cell->__chunk;
     *p = '\0';
 
-    /* FIXME-prepare-support: we should pass prepare value. */
+    /* FIXME-PM-1524: we should pass prepare value. */
```
Comment (quchenhao) [inline src/include/cell.i:473]: @agorrod  told me before PM tickets are not publicly visible in Jira and we should avoid mentioning them in the code. May be double check with @bvpvamsikrishna @agorrod what should be used here.

Comment (tetsuo-cpp) [reply src/include/cell.i:473]: Understood. Thanks @quchenhao.

@agorrod @bvpvamsikrishna 
I get the reasoning but I still think it'd be really useful to tag FIXMEs with the associated project/ticket. That way, we can make it a standard practice to grep the codebase when we're about to close a project and ensure that we've tied up all loose ends.

Comment (agorrod) [reply src/include/cell.i:473]: It's better if we can use public (WT) tickets in the code - can we do that and link the WT ticket to the project - then JIRA manages the visibility for us.

Comment (tetsuo-cpp) [reply src/include/cell.i]: 8888991

**Pull request merged**